exercises/practice/dnd-character/.docs/instructions.md

```diff
@@ -0,0 +1,31 @@
+# Instructions
+
+For a game of [Dungeons & Dragons][dnd], each player starts by generating a character they can play with.
+This character has, among other things, six abilities; strength, dexterity, constitution, intelligence, wisdom and charisma.
```

Comment:
```suggestion
This character has, among other things, six attributes; strength, dexterity, constitution, intelligence, wisdom and charisma.
```

Reply:
This is coming from problem-specifications, so we could lead on the change for the exercise, or customize it.

Not important enough, since I am only noticing it now, and the exercise was started 3 years ago.

Reply:
I'm not too familiar with D&D, but as a long-time player of computer RPGs, attributes would make a lot of sense as a generic term. However, this exercise's story is strongly tied to D&D so I think abilities would be correct here. Ability scores are just the numbers representing the amount of ability. So you could have a high ability score for strength, but strength isn't an ability score but the ability itself.